src/utils/wallet.js

```diff
@@ -100,9 +100,10 @@ class Wallet {
             async signMessage(message, accountId, networkId) {
```

Comment:
@Patrick1904 need to use HD path in getPublicKey above as well

Reply:
@vgrichina please check again after our discussion (only device functions take path, like `client.getPublicKey` or `client.sign`)